docs/Lexicon.md

```diff
@@ -2,16 +2,58 @@
 
 ### Definitions
 
+### A
+
+* ANN
+  Artificial Neural Network.  A computational framework based on a synthetic
```

Comment:
do you intend to render this on a new line. It's not currently, check the "view" button.